tools/schemacode/uv.lock

```diff
@@ -0,0 +1,1353 @@
+version = 1
```

Comment:
this file seems to be not really relevant to the BEP, hence ideally should be introduced separately with description on its function in life